troubadix/troubadix.py

```diff
@@ -148,7 +148,9 @@ def main(args=None):
             )
             sys.exit(1)
         info(f"Start linting {len(parsed_args.files)} files ... ")
-        runner.run(parsed_args.files)
+        # Return exit with 1 if error exist
+        if runner.run(parsed_args.files):
+            sys.exit(1)
```

Comment:
```suggestion
        sys.exit(!runner.run(parsed_args.files))
```

I think this is fitting?

`sys.exit(True)` returns 1, thus we need to negate it ...

Reply:
Personally i think it decreases the readability. I would leave it as it is.

Reply:
ok